resources/todomvc/vanilla-examples/javascript-es5/dist/index.html

```diff
@@ -17,7 +24,7 @@ <h1>todos</h1>
             </header>
             <main class="main">
                 <div class="toggle-all-container">
-                    <input class="toggle-all" type="checkbox">
+                    <input class="toggle-all" type="checkbox" />
```

Comment:
This is unnecessary given this is a HTML file, not XML?

Reply:
that happened when running format - will look into it